src/chariot/core/database/SqliteDatabase.py

```diff
@@ -0,0 +1,62 @@
+import sqlite3
+import os
+import csv
+
+from chariot.core.database.Database import Database
+
+
+class SqliteDatabase(Database):
+    """
+    Implements the Database parent class.
+    Creates a Sqlite database and writes to it.
+
+    Usage:
+    sqliteDB = SqliteDatabase()
+    sqliteDB.insertRow(1000, "json")
+    """
+
+    def __init__(self, db_path: str = 'database.db', flush: bool = False):
+        Database.__init__(self, db_path=db_path, flush=flush)
+        self.connectToDB()
+        if self.flush:
+            self.flushDB()
+
+    def initializeDB(self):
```

Comment:
I think there should be some return value from this method to indicate that the connection was successful.